server/sonar-server/src/main/java/org/sonar/server/user/index/UserIndexDefinition.java

```diff
@@ -54,7 +54,7 @@ public UserIndexDefinition(Settings settings) {
   public void define(IndexDefinitionContext context) {
     NewIndex index = context.create(INDEX);
 
-    index.configureShards(settings);
+    index.configureShards(settings, 1);
```

Comment:
I believe a comment explaining why this value would be convenient for future readers
(and probably using a constant, this really looks like a magic number)